src/services/chapter.ts

```diff
@@ -140,7 +138,7 @@ export class ChapterService {
             { fetchToken: true, ignoreStatus: true },
         );
 
-        return new RoyalResponse(null);
+        return new RoyalResponse("Comment posted");
```

Comment:
i feel like `RoyalResponse` with data should be reserved for cases where the data is actually meaningful

if you really wanted to you could create a `RoyalSuccess` or something to make it clear that no actual data is being returned here